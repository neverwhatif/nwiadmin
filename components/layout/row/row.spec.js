import React from 'react';
import { shallow } from 'enzyme';

import Row from './index';

describe('Row', () => {
    const component = shallow(<Row><p /></Row>);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
    it('should render children', () => {
        expect(component.find('p').length).toBe(1);
    });
});
