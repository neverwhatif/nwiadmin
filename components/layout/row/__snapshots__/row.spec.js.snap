// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Row>
    <p />
  </Row>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p />,
      "className": "root",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p />,
        "className": "root",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
